parser/parser.y

```diff
@@ -6902,6 +6904,15 @@ LoadDataStmt:
 		$$ = x
 	}
 
+IgnoreLines:
+    {
+        $$ = uint64(0)
+    }
+|   "IGNORE" NUM "LINES"
```

Comment:
In MySQL, a negative number will casue syntax error. Could you add a test case about this?

> mysql> load data local infile '/tmp/t.csv' into table t ignore -1 lines;
ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '-1 lines' at line 1
mysql> load data local infile '/tmp/t.csv' into table t ignore 1 lines;
ERROR 1148 (42000): The used command is not allowed with this MySQL version
mysql> quit

Reply:
Got it.